database/migrations/2024_12_10_014534_create_participant_individualized_service_plan.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('participant_individualized_service_plan', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('participant_id')->constrained();
+            $table->string('participant_name');
+            $table->string('client_number');
+            $table->date('review_date');
+            $table->boolean('parenting_skill_development')->default(false);
```

Comment:
Let's ask Good Dads:  Are the service areas pre-filled?  Do they ever get updated?

Just in case, let's update these to _is_service_area

Reply:
For now, we may want the defaults to be true for all minus domestic violence